torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -13,14 +13,9 @@
 
 aten = torch.ops.aten
 
-
-def ref_mm_plus_mm(a, b, c, d, out):
```

Comment:
Would be nice if we had a late way of make_fx/decomposing this and then codegenning to avoid having to add custom op here